flink-clients/src/main/java/org/apache/flink/client/cli/CliFrontend.java

```diff
@@ -1126,7 +1126,7 @@ public static void setJobManagerAddressInConfig(Configuration config, InetSocket
 	 * @return JobGraph extracted from the PackagedProgram
 	 * @throws ProgramInvocationException if the JobGraph generation failed
 	 */
-	private static JobGraph createJobGraph(Configuration configuration, PackagedProgram packagedProgram, int defaultParallelism) throws ProgramInvocationException {
+	public static JobGraph createJobGraph(Configuration configuration, PackagedProgram packagedProgram, int defaultParallelism) throws ProgramInvocationException {
```

Comment:
Maybe move method to utils class.

Reply:
You're right, we should move it to a utility class.